docs/en/build.md

```diff
@@ -55,6 +55,8 @@ Then, follow the steps below to set up the compilation environment:
   ```
 - install [nccl](https://docs.nvidia.com/deeplearning/nccl/install-guide/index.html), and set environment variables:
   ```shell
+  git clone https://github.com/NVIDIA/nccl.git
```

Comment:
I think building `nccl` from source is not a good idea since there are prebuilt libraries.

Reply:
> I don't think building `nccl` from source is a good idea since there are prebuilt libraries. Do you have any trouble installing `nccl` directly? If so, we can discuss about it.

Directly install pre-compiled nccl is also is also possible, but previous environments NCCL_ROOT_DIR and LD_LIBRARY_PATH seem to use the source-compiled nccl (since the pre-compiled nccl doesn't contain a build folder). 
As a result, if I use pre-compiled nccl, the enviroments of nccl should be modified. I'll be submitting a version with pre-compiled nccl later.

Reply:
> I don't think building `nccl` from source is a good idea since there are prebuilt libraries. Do you have any trouble installing `nccl` directly? If so, we can discuss about it.

I have submitted a new version of document. Perhaps the author of the original documentation used the source compilation. So the environment variable needs to include the build folder, but pre-compiled nccl does not include the build folder. So simply removing build in NCCL_ROOT_DIR and NCCL_LIBRARIES  will work!

Reply:
> So the environment variable needs to include the build folder, but pre-compiled nccl does not include the build folder. So simply removing build in NCCL_ROOT_DIR and NCCL_LIBRARIES will work!

ref to https://github.com/InternLM/lmdeploy/blob/8b1b5092d92721f32821b2237ed2b99feba37ac1/builder/manywheel/scripts/install_cuda.sh#L25-L33